src/containers/ModelPanel/index.tsx

```diff
@@ -114,11 +130,47 @@ const ModelPanel: React.FC<ModelPanelProps> = ({
         ),
     };
 
+    const handleRestorDefaultClick = () => {
+        clearUserSelectedColors();
+        setCurrentColorSettings({
+            currentColorSettings: ColorSettings.Default,
+        });
+    };
+
+    const handleRestoreDefaultMouseEnter = () => {
+        if (!defaultUiSettingsApplied) {
+            setCurrentColorSettings({
+                currentColorSettings: ColorSettings.Default,
+            });
+        }
+    };
+
+    const handleRestoreDefaultMouseLeave = () => {
+        if (!defaultUiSettingsApplied) {
+            setCurrentColorSettings({
+                currentColorSettings: ColorSettings.UserSelected,
+            });
+        }
+    };
+
```

Comment:
Hovering previews the default color settings, clicking the button applies them and deletes the old user selected settings.